src/Elastic.CommonSchema/Location.cs

```diff
@@ -1,3 +1,20 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
We can use the condensed header here (3 lines)

++ For starting including though